### PR TITLE
Access Request Enhancements

### PR DESCRIPTION
**Pull request by chowington**
Works toward https://github.com/VEuPathDB/web-eda/issues/1150

Should be tested along with https://github.com/VEuPathDB/ClinEpiWebsite/pull/76

### CODE REVIEW

**Comment by chowington** [comment]
I'm opening this PR for review. It should be noted that most of the additions of `priorAuth` in the PR are just from me adding it to anywhere that had the other terms too. So I don't know whether all of those additions are necessary or desired, but I don't know the purposes of all of those code sections or where they are used. However adding all of them did fix the bug I was encountering. It would be great if someone with more knowledge of this part of the code would investigate more closely!

**Comment by dmfalke** [comment]
> I'm opening this PR for review. It should be noted that most of the additions of `priorAuth` in the PR are just from me adding it to anywhere that had the other terms too. So I don't know whether all of those additions are necessary or desired, but I don't know the purposes of all of those code sections or where they are used. However adding all of them did fix the bug I was encountering. It would be great if someone with more knowledge of this part of the code would investigate more closely!

I'm not too familiar with this part of the code. What is an example of a `priorAuth` value?

**Comment by chowington** [comment]
A `priorAuth` value is basically a long-form string answer to the question, "Did you get approval from the study provider to download this data?" Does that answer your question, or were you asking something else?

**Comment by dmfalke** [comment]
> A `priorAuth` value is basically a long-form string answer to the question, "Did you get approval from the study provider to download this data?" Does that answer your question, or were you asking something else?

Yes, that is helpful! If I understand this PR, that value is being added to a table(s) that end users see (the hooks are used in https://github.com/VEuPathDB/web-study-data-access/blob/main/src/study-access/components/StudyAccessController.tsx.

I haven't yet tested this PR, but it seems like we should get a screenshot of the result and ask the clinepi team if they want this addition, unless of course they asked for it. Does that sound reasonable to you?

File: src/study-access/studyAccessHooks.tsx [hunk 1]
```diff
@@ -1421,19 +1431,21 @@ function makeContentSearchableSring(
     analysisPlan && 'Analysis Plan:',
     analysisPlan,
     disseminationPlan && 'Dissemination Plan:',
-    disseminationPlan
+    disseminationPlan,
+    priorAuth && 'Prior Authorization:',
```
Comment (dmfalke) [inline src/study-access/studyAccessHooks.tsx:1435]: Does the value of `priorAuth` need to be added here?
```suggestion
    priorAuth && 'Prior Authorization:',
    priorAuth,
```

**Review by dmfalke — APPROVED** [review]
This all looks good to me. I did add one more comment related to `makeContentSearchableString`.

**Pull request merged**